src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -259,4 +260,90 @@ public void systemEndsPatientVisit() {
 		assertNull(visitsDashboardPage.getActiveVisit());
 		dashboardPage = visitsDashboardPage.goToPatientDashboard();
 	}
+
+	@When("a user clicks on Allergy2 link from Patient dashboard page2")
+	public void loadManageAllergy2() {
+		allergyPage = (AllergyPage) dashboardPage.clickOnAllergiesWidgetLink()
+				.waitForPage();
+	}
+
+	@Then("the system loads Allergy2 board page")
+	public void loadsManageAllergy2() {
+				assertTrue(textExists("Allergies"));
+	}
```

Comment:
Use the method [systemLoadsAllergiesPage()](https://github.com/openmrs/openmrs-contrib-qaframework/blob/2181d1a931c69a264d3c4c6e7e159d4ecf60f9aa/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java#L119) rather

Reply:
@Ndacyayisenga-droid refer to my comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/137/files#r723235027)!